src/graphics/Screen.cpp

```diff
@@ -1596,6 +1595,112 @@ static void drawNodeInfo(OLEDDisplay *display, OLEDDisplayUiState *state, int16_
     drawColumns(display, x, y, fields);
 }
 
+/// Draw the last waypoint we received
+static void drawWaypointFrame(OLEDDisplay *display, OLEDDisplayUiState *state, int16_t x, int16_t y)
```

Comment:
What would it take to move this into WaypointModule.drawFrame()?

Reply:
It'd be nice to have it contained like that. Honestly, I just modified the existing `drawWaypointFrame` method right where it was without giving it too much thought. 

> What would it take

A bunch of graphics stuff seems to lean on static functions inside of Screen.cpp, but outside the screen class. I don't think any of the frames drawn directly by modules can make use of these functions right now. Exposing those tidily to the mesh module classes would probably be the main thing to deal with.

Reply:
This is going to take a bit more untangling. I don't think the PR should wait on it.